app/services/allocation/AssessorAllocationService.scala

```diff
@@ -16,31 +16,50 @@
 
 package services.allocation
 
+import connectors.{ AuthProviderClient, CSREmailClient, EmailClient }
 import model.Exceptions.OptimisticLockException
+import model._
+import model.command.CandidateAllocation
 import model.exchange.{ EventAssessorAllocationsSummaryPerSkill, EventWithAllocationsSummary }
-import model.{ AllocationStatuses, command, exchange, persisted }
 import model.persisted.eventschedules.EventType.EventType
 import model.persisted.eventschedules.Venue
+import model.stc.EmailEvents.{ CandidateAllocationConfirmed, CandidateAllocationRequestConfirmation }
+import model.stc.StcEventTypes.StcEvents
+import play.api.mvc.RequestHeader
+import repositories.application.GeneralApplicationRepository
 import repositories.{ AssessorAllocationMongoRepository, CandidateAllocationMongoRepository }
 import services.events.EventsService
+import services.stc.{ EventSink, StcEventService }
+import uk.gov.hmrc.play.http.HeaderCarrier
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 object AssessorAllocationService extends AssessorAllocationService {
-  def allocationRepo = repositories.assessorAllocationRepository
-  override val eventsService = EventsService
-  def candidateAllocationRepo = repositories.candidateAllocationRepository
+  val allocationRepo: AssessorAllocationMongoRepository = repositories.assessorAllocationRepository
```

Comment:
To distinguish clearly from candidateAllocationRepo, I would call it assessorAllocationRepo